docs/operations/etcd_backup_restore_encryption.md

```diff
@@ -55,22 +55,45 @@ etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/e
 Add a restore command for both clusters:
 
 ```
-etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/main restore-backup [main backup file]
-etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/events restore-backup [events backup file]
+etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/main restore-backup [main backup dir]
+etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/events restore-backup [events backup dir]
 ```
 
 Note that this does not start the restore immediately; you need to restart etcd on all masters.
 You can do this with a `docker stop` or `kill` on the etcd-manager containers on the masters (the container names start with `k8s_etcd-manager_etcd-manager`).
 The etcd-manager containers should restart automatically, and pick up the restore command. You also have the option to roll your masters quickly, but restarting the containers is preferred.
- 
-A new etcd cluster will be created and the backup will be 
-restored onto this new cluster. Please note that this process might take a short while, 
-depending on the size of your cluster. 
+
+A new etcd cluster will be created and the backup will be
+restored onto this new cluster. Please note that this process might take a short while,
+depending on the size of your cluster.
 
 You can follow the progress by reading the etcd logs (`/var/log/etcd(-events).log`)
 on the master that is the leader of the cluster (you can find this out by checking the etcd logs on all masters).
 Note that the leader might be different for the `main` and `events` clusters.
 
+After the restore, you will probably face an intermitent connection to apiserver.
```

Comment:
Looks like the Travis CI job caught this

```suggestion
After the restore, you will probably face an intermittent connection to apiserver.
```

Reply:
Done!